pupil-spa/src/app/connectivity-error/connectivity-error.component.html

```diff
@@ -0,0 +1,57 @@
+<app-header class="full-width"></app-header>
+<main id="content" class="js-content" role="main">
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <header class="page-header">
+        <h1 tabindex="0">Loading error</h1>
+      </header>
+      <br />
+      <div class="notice">
+        <strong class="visually-hidden">Warning</strong>
+        <app-svg-warning></app-svg-warning>
+        <strong>
+          Please tell your teacher
+        </strong>
+      </div>
+      <br />
+      <details>
+        <summary><span class="summary">Instructions for teacher</span></summary>
```

Comment:
As per all the recent copy changes I suggest this should be 'Instructions for teachers'

Reply:
@jon-shipley done